lua/weapons/weapon_ttt_rifle_stn.lua

```diff
@@ -0,0 +1,230 @@
+-- Traq Rifle created by https://steamcommunity.com/profiles/76561198813867340
+
+include("autorun/nl_library.lua")
+
+if SERVER then
+    AddCSLuaFile()
+end
+
+DEFINE_BASECLASS("weapon_tttbase")
+
+SWEP.HoldType = "ar2"
+
+if CLIENT then
+    SWEP.PrintName = "Rifle (STN)"
+    SWEP.Slot = 6
+
+    SWEP.ViewModelFlip = false
+    SWEP.ViewModelFOV = 54
+    
+    SWEP.EquipMenuData = {
+        type = "item_weapon",
+	name = "Rifle (STN)",
+        desc = "A Non-Lethal Rifle.\n\nHeadshots deal 200 stun for 20 seconds.\n\nBodyshots deal 50 stun for 15 seconds."
+    }
+    
+    SWEP.Icon = "vgui/ttt/icon_scout"
+    SWEP.IconLetter = "n"
+end
+
+SWEP.Base = "weapon_tttbase"
+
+SWEP.Kind = WEAPON_EQUIP
+SWEP.WeaponID = AMMO_RIFLE
+SWEP.CanBuy = { ROLE_DETECTIVE , ROLE_TRAITOR }
+SWEP.LimitedStock = false
+SWEP.builtin = true
+SWEP.spawnType = WEAPON_TYPE_SNIPER
```

Comment:
Same as what I mentioned on the M16 - remove `SWEP.builtin` and reconsider `SWEP.spawnType`